### PR TITLE
Upload API - final changes

### PR DESCRIPTION
**Pull request by ryangec**
[Link to orignal Wix Docs PR](https://github.com/wix-incubator/wix-code-docs/pull/2949)

### CODE REVIEW

File: wix-media-v2/wix-media-v2/Files/guides/resumable-upload-api.md [hunk 1]
```diff
@@ -112,8 +112,15 @@ async function resumableFileUpload(resumableUploadUrlResponse: GenerateFileResum
 }
 ```
 
-
 ## Status/Error Codes
 Errors from this endpoint will include an HTTP status code.
         
+### Use uploaded media on your site
+After making the REST HTTP request, you can either display the media on your site directly from the [Media Manager](https://support.wix.com/en/article/wix-media-about-the-media-manager#accessing-the-media-manager-1), or you can set the uploaded media to `$w` frontend elements programatically.
```
Comment (imnotashrimp) [inline wix-media-v2/wix-media-v2/Files/guides/resumable-upload-api.md:119]:
```suggestion
After making the REST HTTP request, you can either display the media on your site directly from the [Media Manager](https://support.wix.com/en/article/wix-media-about-the-media-manager#accessing-the-media-manager-1), or you can set the uploaded media to `$w` frontend elements.
```

File: wix-media-v2/wix-media-v2/Files/guides/resumable-upload-api.md [hunk 2]
```diff
@@ -112,8 +112,15 @@ async function resumableFileUpload(resumableUploadUrlResponse: GenerateFileResum
...
+
+In order to display the media files on the frontend of your site with code, you first need the file's Media Manager URL.
```
Comment (imnotashrimp) [inline wix-media-v2/wix-media-v2/Files/guides/resumable-upload-api.md:121]:
```suggestion
To display the media files on the frontend of your site with code, you first need to retrieve the file's Media Manager URL.
```

File: wix-media-v2/wix-media-v2/Files/guides/resumable-upload-api.md [hunk 3]
```diff
@@ -112,8 +112,15 @@ async function resumableFileUpload(resumableUploadUrlResponse: GenerateFileResum
...
 
+To retrieve the Media Manager URL and update your frontend elements:
```
Comment (imnotashrimp) [inline wix-media-v2/wix-media-v2/Files/guides/resumable-upload-api.md:123]:
```suggestion
To retrieve the Media Manager URL:
```

File: wix-media-v2/wix-media-v2/Files/guides/resumable-upload-api.md [hunk 4]
```diff
@@ -112,8 +112,15 @@ async function resumableFileUpload(resumableUploadUrlResponse: GenerateFileResum
...
+1.  Call [`getFileDescriptor()`](https://www.wix.com/velo/reference/wix-media-v2/files/getfiledescriptor), passing in `file.id` from the Upload API's [response object](#Response) as its only parameter. 
+2. Retrieve the valid Media Manager URL from the `media` property in the `FileDescriptor` object that was returned in the previous step.
+3. Set the value of the associated `media` property to the [`src`](https://www.wix.com/velo/reference/$w/image/src) property in your site's `$w` frontend elements.
```
Comment (imnotashrimp) [inline wix-media-v2/wix-media-v2/Files/guides/resumable-upload-api.md:126]:
```suggestion
3. To use the media in a `$w` element, set the value of the associated `media` property to the element's `src` property. 
```

**Review by imnotashrimp — APPROVED** [review]

**Comment by ryangec** [comment]
#skipreview

**Pull request merged**